### ISSUE TITLE
vite-plugin v4.0.2 does not work with Remix

### ISSUE THREAD

**Issue by MattyBalaam** [vanilla-extract-css/vanilla-extract#1311]
### Describe the bug

Trying to use 4.0.2 with remix throws an error "The Remix Vite plugin requires the use of a Vite config file"

I have seen the issue with yarn 1, yarn 4 (where I first encountered as part of a monorepo) and npm

<img width="1137" alt="Screenshot 2024-02-04 at 10 46 10" src="https://github.com/vanilla-extract-css/vanilla-extract/assets/1246923/62253814-ff99-48a0-abe6-ffbde72fd2f9">

I have a reproduction - just install and run the dev script

It works if you revert back to 4.0.0 

### Reproduction

https://github.com/MattyBalaam/remix-vanilla-4.0.2-issue

### System Info

```shell
System:
    OS: macOS 14.3
    CPU: (11) arm64 Apple M3 Pro
    Memory: 86.91 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.18.2 - /opt/homebrew/opt/node@18/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 9.8.1 - /opt/homebrew/opt/node@18/bin/npm
    pnpm: 8.15.1 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 121.0.6167.139
    Safari: 17.3
  npmPackages:
    @vanilla-extract/css: 1.14.1 => 1.14.1 
    @vanilla-extract/vite-plugin: 4.0.2 => 4.0.2
```

### Used Package Manager

npm

### Validations

- [X] Check that there isn't [already an issue](https://github.com/vanilla-extract-css/vanilla-extract/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by askoufis** [comment]
This was fixed in #1308, which should be released soon.

**Comment by mrm007** [comment]
This should be fixed in v4.0.3

**Comment by MattyBalaam** [comment]
It’s working here for me. Many thanks for super fast fix.
